finance/proposal-calls/cycle3/cycle3.md

```diff
@@ -0,0 +1,90 @@
+# Cycle III funding requests and ideas
+
+
+# Changes relative to the previous cycle
+
+This cycle includes funding from both NASA ROSES and Moore Foundation. Process-wise, most of the below is similar to the Cycle II (at the time called “Phase II”), with some additional context and minor practical changes based on how Phase II progressed. The main difference in this cycle is the “hiring process” mentioned below. Specifically, in this cycle **_you can now request funding for a project that you do not necessarily wish to execute yourself, but that you would like Astropy to find someone to do the work._** We hope this will serve both expand our community and get desired work done that our current community does not have the capacity to execute.  
```

Comment:
```suggestion
This cycle includes funding from both NASA ROSES and Moore Foundation. Process-wise, most of the below is similar to the Cycle II (at the time called “Phase II”), with some additional context and minor practical changes based on how Phase II progressed. The main difference in this cycle is the “hiring process” mentioned below. Specifically, in this cycle **_you can now request funding for a project that you do not necessarily wish to execute yourself, but that you would like Astropy to find someone to do the work._** We hope this will both expand our community and get desired work done that our current community does not have the capacity to execute.  
```